http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -621,12 +679,16 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
     deletion.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
   }
 
-  for (appType <- List(AppType.Wds, AppType.Cromwell, AppType.HailBatch, AppType.WorkflowsApp))
+  for (
```

Comment:
CromwellRunnerApp shouldn't be created with WorkspaceShared access, so I'm not sure if these tests should be altered since both this test and the following one create the app with the shared access.

Reply:
Ah I see. Yeah I can create a separate test for Cromwell Runner app then. The following test seems to be creating apps with WSM resources which is right for Cromwell Runner app too no?